### ISSUE TITLE
Unexpected behavior in plotting with filter

### ISSUE THREAD

**Issue by rohrlich** [emer/etable#38]
It appears that checking or unchecking any variable clears the filter, took a couple of minutes to realize this. Makes sense to me to leave the filter in place until the user clicks "unfilter".
Also, suggest that be an obvious indicator when the results are filtered to make the state transparent.

**Comment by rohrlich** [comment]
Plus, as it is, I need to refilter every time I change what I'm plotting.

**Comment by rcoreilly** [comment]
yeah we need to save and restore the filter across updates.  current version is a "quick and dirty" thing that works after graph was built.

**Comment by rohrlich** [comment]
Also, filtering should maintain while running.
